packages/docs/site/docs/blueprints/08-examples.md

```diff
@@ -85,6 +85,37 @@ blueprint={{
 		]
 }} />
 
+## How to work with WP-CLI on Playground
+
+To run WP-CLI commands on a Playground instance, you need to set up two things: mount the `/wordpress/` directory to your local filesystem and ensure the SQLite database integration is correctly configured. These steps allow WP-CLI to recognize the site as a valid WordPress installation and connect to its database.
```

Comment:
Note you can run WP-CLI commands via a special Blueprint step. This section is specific to running WP-CLI commands from your terminal when the WordPress site was created via Playground CLI.